--- build/openni/Include//XnBaseNode.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnBaseNode.h	2010-12-05 21:47:01.000000000 -0800
@@ -28,7 +28,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnDataTypes.h>
+#include <ni/XnDataTypes.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnCodecIDs.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnCodecIDs.h	2010-12-05 21:47:01.000000000 -0800
@@ -24,7 +24,7 @@
 #ifndef __NICODECIDS_H__
 #define __NICODECIDS_H__
 
-#include "XnTypes.h"
+#include "ni/XnTypes.h"
 
 #define XN_CODEC_NULL				XN_CODEC_ID(0, 0, 0, 0)
 #define XN_CODEC_UNCOMPRESSED		XN_CODEC_ID('N','O','N','E')
--- build/openni/Include//XnContext.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnContext.h	2010-12-05 21:47:01.000000000 -0800
@@ -82,9 +82,9 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
-#include <XnQueries.h>
-#include <XnPrdNodeInfoList.h>
+#include <ni/XnTypes.h>
+#include <ni/XnQueries.h>
+#include <ni/XnPrdNodeInfoList.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnCppWrapper.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnCppWrapper.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,8 +29,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnOpenNI.h>
-#include <XnCodecIDs.h>
+#include <ni/XnOpenNI.h>
+#include <ni/XnCodecIDs.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnDataTypes.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnDataTypes.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnOS.h"
+#include "ni/XnOS.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnEnumerationErrors.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnEnumerationErrors.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 /**
  * @ingroup cref
--- build/openni/Include//XnEvent.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnEvent.h	2010-12-05 21:47:01.000000000 -0800
@@ -28,10 +28,10 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnCallback.h"
-#include "XnList.h"
-#include "XnTypes.h"
-#include "XnOSCpp.h"
+#include "ni/XnCallback.h"
+#include "ni/XnList.h"
+#include "ni/XnTypes.h"
+#include "ni/XnOSCpp.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnFPSCalculator.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnFPSCalculator.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,8 +29,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnPlatform.h>
-#include <XnStatus.h>
+#include <ni/XnPlatform.h>
+#include <ni/XnStatus.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnGeneralBuffer.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnGeneralBuffer.h	2010-12-05 21:47:01.000000000 -0800
@@ -28,9 +28,9 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnPlatform.h"
-#include "XnOS.h"
-#include "XnStatusCodes.h"
+#include "ni/XnPlatform.h"
+#include "ni/XnOS.h"
+#include "ni/XnStatusCodes.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnHash.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnHash.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnList.h"
+#include "ni/XnList.h"
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnLicensing.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnLicensing.h	2010-12-05 21:47:01.000000000 -0800
@@ -36,7 +36,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnList.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnList.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,10 +29,10 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnDataTypes.h>
-#include <IXnNodeAllocator.h>
-#include <XnNodeAllocator.h>
-#include <XnNode.h>
+#include <ni/XnDataTypes.h>
+#include <ni/IXnNodeAllocator.h>
+#include <ni/XnNodeAllocator.h>
+#include <ni/XnNode.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnLog.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnLog.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnOS.h"
+#include "ni/XnOS.h"
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnMacros.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnMacros.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnStatusCodes.h"
+#include "ni/XnStatusCodes.h"
 
 //---------------------------------------------------------------------------
 // Macros
--- build/openni/Include//XnModuleCppInterface.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnModuleCppInterface.h	2010-12-05 21:47:01.000000000 -0800
@@ -35,9 +35,9 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnModuleInterface.h"
-#include "XnList.h"
-#include "XnCppWrapper.h"
+#include "ni/XnModuleInterface.h"
+#include "ni/XnList.h"
+#include "ni/XnCppWrapper.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnModuleCppRegistratration.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnModuleCppRegistratration.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnModuleCppInterface.h"
+#include "ni/XnModuleCppInterface.h"
 
 using namespace xn;
 
--- build/openni/Include//XnModuleInterface.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnModuleInterface.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnNodeAllocator.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnNodeAllocator.h	2010-12-05 21:47:01.000000000 -0800
@@ -25,8 +25,8 @@
 #ifndef _NISIMPLENODEALLOCATOR_H
 #define _NISIMPLENODEALLOCATOR_H
 
-#include <IXnNodeAllocator.h>
-#include <XnNode.h>
+#include <ni/IXnNodeAllocator.h>
+#include <ni/XnNode.h>
 
 class XnNodeAllocator : public INiNodeAllocator
 {
--- build/openni/Include//XnNode.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnNode.h	2010-12-05 21:47:01.000000000 -0800
@@ -29,8 +29,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnDataTypes.h>
-#include <XnBaseNode.h>
+#include <ni/XnDataTypes.h>
+#include <ni/XnBaseNode.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnOpenNI.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnOpenNI.h	2010-12-05 21:47:01.000000000 -0800
@@ -28,18 +28,18 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnTypes.h"
-#include "XnContext.h"
-#include "XnLicensing.h"
-#include "XnUtils.h"
-#include "XnPrdNodeInfo.h"
-#include "XnQueries.h"
-#include "XnPrdNode.h"
-#include "XnEnumerationErrors.h"
+#include "ni/XnTypes.h"
+#include "ni/XnContext.h"
+#include "ni/XnLicensing.h"
+#include "ni/XnUtils.h"
+#include "ni/XnPrdNodeInfo.h"
+#include "ni/XnQueries.h"
+#include "ni/XnPrdNode.h"
+#include "ni/XnEnumerationErrors.h"
 
-#include "XnVersion.h"
-#include "XnStatusCodes.h"
-#include "XnStatus.h"
+#include "ni/XnVersion.h"
+#include "ni/XnStatusCodes.h"
+#include "ni/XnStatus.h"
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnOSCpp.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnOSCpp.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnOS.h>
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnOS.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnOS.h	2010-12-05 21:49:27.000000000 -0800
@@ -29,9 +29,9 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnPlatform.h"
-#include "XnMacros.h"
-#include "XnStatusCodes.h"
+#include "ni/XnPlatform.h"
+#include "ni/XnMacros.h"
+#include "ni/XnStatusCodes.h"
 
 //---------------------------------------------------------------------------
 // Defines
@@ -45,9 +45,9 @@
 // OS Identifier 
 //---------------------------------------------------------------------------
 #if defined(_WIN32)
-	#include "Win32/XnOSWin32.h"
+	#include "ni/Win32/XnOSWin32.h"
 #elif defined(linux)
-	#include "Linux-x86/XnOSLinux-x86.h"
+	#include "ni/Linux-x86/XnOSLinux-x86.h"
 #endif
 
 //---------------------------------------------------------------------------
Only in include/ni: XnOS.h~
--- build/openni/Include//XnPlatform.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnPlatform.h	2010-12-05 21:47:02.000000000 -0800
@@ -61,11 +61,11 @@
 		#endif
 	#endif
 
-	#include "Win32/XnPlatformWin32.h"
+	#include "ni/Win32/XnPlatformWin32.h"
 #elif (linux && (i386 || __x86_64__))
-	#include "Linux-x86/XnPlatformLinux-x86.h"
+	#include "ni/Linux-x86/XnPlatformLinux-x86.h"
 #elif (linux && __arm__)
-	#include "Linux-Arm/XnPlatformLinux-Arm.h"
+	#include "ni/Linux-Arm/XnPlatformLinux-Arm.h"
 #else
 	#error OpenNI Platform Abstraction Layer - Unsupported Platform!
 #endif
--- build/openni/Include//XnPrdNode.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnPrdNode.h	2010-12-05 21:47:02.000000000 -0800
@@ -35,7 +35,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnPrdNodeInfo.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnPrdNodeInfo.h	2010-12-05 21:47:02.000000000 -0800
@@ -37,7 +37,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnPrdNodeInfoList.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnPrdNodeInfoList.h	2010-12-05 21:47:02.000000000 -0800
@@ -82,7 +82,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnProfiling.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnProfiling.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnOS.h>
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnQueries.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnQueries.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnTypes.h"
+#include "ni/XnTypes.h"
 
 //---------------------------------------------------------------------------
 // Exported Functions
--- build/openni/Include//XnQueue.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnQueue.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnList.h"
+#include "ni/XnList.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnScheduler.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnScheduler.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnOS.h>
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnStack.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnStack.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnList.h"
+#include "ni/XnList.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnStatusCodes.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnStatusCodes.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnStatus.h"
+#include "ni/XnStatus.h"
 
 //---------------------------------------------------------------------------
 // Status Messages
--- build/openni/Include//XnStatus.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnStatus.h	2010-12-05 21:47:02.000000000 -0800
@@ -26,7 +26,7 @@
 #ifndef __XN_OPEN_NITE_STATUS_H__
 #define __XN_OPEN_NITE_STATUS_H__
 
-#include "XnPlatform.h"
+#include "ni/XnPlatform.h"
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnStatusRegister.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnStatusRegister.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnStatus.h"
+#include "ni/XnStatus.h"
 
 //---------------------------------------------------------------------------
 // Structs
--- build/openni/Include//XnStringsHash.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnStringsHash.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,8 +29,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnHash.h"
-#include <XnOS.h>
+#include "ni/XnHash.h"
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnThreadSafeQueue.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnThreadSafeQueue.h	2010-12-05 21:47:02.000000000 -0800
@@ -28,8 +28,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnQueue.h>
-#include <XnOS.h>
+#include <ni/XnQueue.h>
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Types
--- build/openni/Include//XnTypes.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnTypes.h	2010-12-05 21:47:02.000000000 -0800
@@ -28,8 +28,8 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnStatus.h>
-#include <XnOS.h>
+#include <ni/XnStatus.h>
+#include <ni/XnOS.h>
 
 //---------------------------------------------------------------------------
 // Defines
--- build/openni/Include//XnUSB.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnUSB.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnOS.h"
+#include "ni/XnOS.h"
 
 //---------------------------------------------------------------------------
 // Structures & Enums
--- build/openni/Include//XnUtils.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnUtils.h	2010-12-05 21:47:02.000000000 -0800
@@ -36,7 +36,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include <XnTypes.h>
+#include <ni/XnTypes.h>
 
 //---------------------------------------------------------------------------
 // Functions
--- build/openni/Include//XnVersion.h	2010-12-05 21:08:00.000000000 -0800
+++ include/ni/XnVersion.h	2010-12-05 21:47:02.000000000 -0800
@@ -29,7 +29,7 @@
 //---------------------------------------------------------------------------
 // Includes
 //---------------------------------------------------------------------------
-#include "XnPlatform.h"
+#include "ni/XnPlatform.h"
 
 //---------------------------------------------------------------------------
 // Defines
