--- Platform/Linux-x86/CreateRedist/Redist_OpenNi.py.orig	2010-12-05 20:57:34.000000000 -0800
+++ Platform/Linux-x86/CreateRedist/Redist_OpenNi.py	2010-12-05 20:57:47.000000000 -0800
@@ -229,10 +229,10 @@
 shutil.copy("../../LGPL.txt", "Redist")
 
 #lib
-shutil.copy("Bin/Release/libnimCodecs.so", "Redist/Lib")
-shutil.copy("Bin/Release/libnimMockNodes.so", "Redist/Lib")
-shutil.copy("Bin/Release/libnimRecorder.so", "Redist/Lib")
-shutil.copy("Bin/Release/libOpenNI.so", "Redist/Lib")
+shutil.copy("Lib/Release/libnimCodecs.so", "Redist/Lib")
+shutil.copy("Lib/Release/libnimMockNodes.so", "Redist/Lib")
+shutil.copy("Lib/Release/libnimRecorder.so", "Redist/Lib")
+shutil.copy("Lib/Release/libOpenNI.so", "Redist/Lib")
 
 #bin
 shutil.copy("Bin/Release/niReg", "Redist/Bin")
